src/Cli/dotnet/commands/dotnet-sln/SlnCommandParser.cs

```diff
@@ -26,7 +29,12 @@ public static CliCommand GetCommand()
 
         private static CliCommand ConstructCommand()
         {
-            DocumentedCommand command = new("sln", DocsLink, LocalizableStrings.AppFullName);
+            DocumentedCommand command = new(CommandName, DocsLink, LocalizableStrings.AppFullName);
+
+            for (int i = 0; i < CommandAliases.Length; i++)
+            {
+                command.Aliases.Add(CommandAliases[i]);
```

Comment:
since there's only one alias, probably could just make the Alias a string and call add to clean this up